modules/cli/src/main/scala/scala/cli/commands/package0/PackageOptions.scala

```diff
@@ -13,15 +13,17 @@ import scala.build.{BuildThreads, Positioned}
 import scala.cli.commands.package0.PackageOptions
 import scala.cli.commands.shared.*
 
-// format: off
-@HelpMessage(
+@HelpMessage({
+  val cmdName = "package"
   s"""Compile and package Scala code.
      |
-     |${HelpMessages.commandConfigurations("package")}
+     |${HelpMessages.commandConfigurations(cmdName)}
```

Comment:
Why not use `Package.name`? The same suggestion for similar cases

Reply:
`Package` is parametrised with `PackageOptions`, so `Package.name` isn't available before `PackageOptions` is loaded.
in other words, that wouldn't compile